src/components/Calculator.js

```diff
@@ -23,7 +23,7 @@ const Calculator = () => {
           {next}
         </div>
         <div className={styles.buttons}>
-          <button type="button" className={styles.btnNumbers} onClick={handleClick}>AC</button>
+          <button type="button" className={styles.btnNumbers} onClick={handleClick}>AC </button>
           <button type="button" className={styles.btnNumbers} onClick={handleClick}>+/-</button>
           <button type="button" className={styles.btnNumbers} onClick={handleClick}>%</button>
           <button type="button" className={`${styles.btnNumbers} ${styles.btnOperation}`} onClick={handleClick}>/</button>
```

Comment:
### App.js 


- Hey, let's also refactor the app.js  [here](https://github.com/M1rn4/math-magician/blob/72fd1947a9e548112c83293ed775fdc209bd53cc/src/App.js#L6-L11)